erpnext/www/book-appointment/index.js

```diff
@@ -0,0 +1,213 @@
+
+frappe.ready(() => {
+    initialise_select_date()
+})
+window.holiday_list = [];
+
+async function initialise_select_date() {
+    navigate_to_page(1);
+    await get_global_variables();
+    setup_date_picker();
+    setup_timezone_selector();
+    hide_next_button();
+}
+
+async function get_global_variables() {
+    window.appointment_settings = (await frappe.call({
```

Comment:
A constructor (Class based) could have made data sharing easier. Perhaps we can save it for the refactor